src/core/ext/transport/binder/client/channel_create_impl.cc

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <grpc/impl/codegen/port_platform.h>
+
+#include "src/core/ext/transport/binder/client/channel_create_impl.h"
+
+#include <memory>
+#include <utility>
+
+#include "src/core/ext/transport/binder/transport/binder_transport.h"
+#include "src/core/ext/transport/binder/wire_format/binder.h"
+#include "src/core/lib/channel/channel_args.h"
+#include "src/core/lib/surface/api_trace.h"
+#include "src/core/lib/surface/channel.h"
+
+namespace grpc_binder {
+namespace internal {
+
+grpc_channel* CreateChannelFromBinderImpl(
+    std::unique_ptr<Binder> endpoint_binder, const grpc_channel_args* args) {
+  grpc_core::ExecCtx exec_ctx;
+  GRPC_API_TRACE("grpc_channel_create_from_binder(target=%p, args=%p)", 2,
```

Comment:
Do we still need this `GRPC_API_TRACE` if we don't have a C API for creating binder channel now? Or should we also add a C API for this?

Reply:
Also, I think it probably makes more sense for this to be in `grpc::internal` instead of `grpc_binder::internal`. What do you think?